test/lib/default-transformer-test.js

```diff
@@ -19,10 +19,10 @@ describe('default-transformer', () => {
     });
 
     it('works in a sentence with multiple tags & a URL', () => {
-        const sentence = `#test sentenc #tag and url: http://docs.panoptes.apiary.io/#reference/user/users-collection/list-all-users`
+        const sentence = `#test sentence #tag and url: http://docs.panoptes.apiary.io/#reference/user/users-collection/list-all-users`
 
         const htmlSentence = replaceSymbols(sentence, {project, baseURI})
-        expect(htmlSentence).to.equal(`#test sentenc #tag and url: http://docs.panoptes.apiary.io/#reference/user/users-collection/list-all-users`)
+        expect(htmlSentence).to.equal(`[#test](/talk/search?query=test) sentence[#tag](/talk/search?query=tag) and url: http://docs.panoptes.apiary.io/#reference/user/users-collection/list-all-users`)
```

Comment:
I don't know how this could've passed before, it was expecting the tags to not be parsed at all